### PR TITLE
 #613 Pivot table Field Settings saving and loading

### PR DESCRIPTION
**Pull request by b0bi79**
Saving and loading pivot table field options.
Issue #613

### CODE REVIEW

**Comment by znakeeye** [comment]
Any plans to include this patch in an upcoming release?

**Comment by igitur** [comment]
Sorry, @b0bi79 , I dropped the ball on this. Would you mind resolving the conflict and pushing? I'll try get it in for v0.92

File: ClosedXML_Examples/app.config [hunk 1]
```diff
@@ -1,6 +1,6 @@
-<?xml version="1.0"?>
-<configuration>
-  <startup>
-    <supportedRuntime version="v4.0" sku=".NETFramework,Version=v4.0"/>
-  </startup>
-</configuration>
+<?xml version="1.0"?>
```
Comment (igitur) [inline ClosedXML_Examples/app.config]: What changed in this file?

File: ClosedXML_Sandbox/app.config [hunk 2]
```diff
@@ -1,6 +1,6 @@
-<?xml version="1.0"?>
-<configuration>
-  <startup>
-    <supportedRuntime version="v4.0" sku=".NETFramework,Version=v4.0"/>
-  </startup>
-</configuration>
+<?xml version="1.0"?>
```
Comment (igitur) [inline ClosedXML_Sandbox/app.config]: This one too...? What changed in this file?

File: ClosedXML_Examples/app.config (hunk 1 above)
Comment (b0bi79) [reply ClosedXML_Examples/app.config]: I do not know, the git showed me that the files were changed and did not allow to roll back the changes.

File: ClosedXML_Sandbox/app.config (hunk 2 above)
Comment (b0bi79) [reply ClosedXML_Sandbox/app.config]: I do not know, the git showed me that the files were changed and did not allow to roll back the changes.

**Comment by igitur** [comment]
Ok, thanks for your contribution.  Please don't disappear! You've become quiet lately ;-)

**Comment by b0bi79** [comment]
Now a lot of work. When I will be more free, I will return.

**Pull request merged**